### DISCUSSION TITLE
Programı kaldırmadan, nerelerde dosyası-klasörü olduğunu gösteren bir özellik var mı?

### DISCUSSION THREAD

**Discussion by Recoo28** [Klocman/Bulk-Crap-Uninstaller#622] [Q&A] [answered]
Let's take BCU as an example. Without uninstalling this program from my computer, is there a feature that shows which drive, where it has files and what it does in the registry? Or do you know of a program that does this?

**Comment by Klocman** [comment, marked as answer]
You can use the "manual uninstall" option, it will show you everything that BCU finds without removing any of it until you confirm.
